src/main/java/com/hyperwallet/clientsdk/util/HyperwalletApiClient.java

```diff
@@ -196,4 +210,22 @@ private String decryptResponse(String responseBody) throws ParseException, IOExc
         return isEncrypted ? hyperwalletEncryption.decrypt(responseBody) : responseBody;
     }
 
+    public WebResource getWebResource(final String url) {
+        client.addFilter(new HTTPBasicAuthFilter(this.username, this.password));
+        return client.resource(url);
+    }
+
+    public <T> T put(final String url, final FormDataMultiPart formDataMultiPart, final Class<T> type) {
+        Response response = new Response();
+        try {
+            webResource = getWebResource(url);
+            ClientResponse clientResponse = webResource.type(MediaType.MULTIPART_FORM_DATA_TYPE).put(ClientResponse.class, formDataMultiPart);
```

Comment:
just like i commented in PR #60 can we make this lightweight? we are importing the whole jersey client just for this `put` but using different method on any other http methods